packages/overlay/stories/overlay-element.stories.ts

```diff
@@ -152,6 +152,45 @@ longpress.args = {
     type: 'auto',
 };
 
+export const transformed = (args: Properties): TemplateResult => html`
+    <style>
+        .transformed {
```

Comment:
Can we use logical CSS here too?
```
.transformed {
    transform: translateX(-50%);
    position: absolute;
    inset-inline-start: 200px; /* Equivalent to left: 200px; */
    inset-inline-end: auto; /* Resets inline-end property */
    inset-block-start: 200px; /* Equivalent to top: 200px; */
    inset-block-end: auto; /* Resets block-end property */
    width: 100px;
    height: 50px;
}
```

Reply:
Not specifically germane to the issue this confirms, but it's good practice to get into. Thanks for the reminder!